docs/contributing/managing-pull-requests.md

```diff
@@ -172,9 +172,9 @@ Every PR opened in the repository needs to be approved before it can be merged.
 Typically this is:
 
 - **gatsbyjs/themes** for certain features like Themes, Recipes, and Admin
-- **gatsbyjs/learning** for documentation
+- **gatsbyjs/documentation** for documentation
 - **gatsbyjs/core** for any other code
-- **gatsbyjs/website** for anything related to GatsbyJS.org
+- **gatsbyjs/website** for anything related to GatsbyJS.com
```

Comment:
Is this group still necessary since website things are now in the other repo?

Reply:
I'm not actually even a part of this team since I think it's pretty old 🤔  my opinion is it could be removed as well since we'd prefer to have site issues about the site get filed through support/Twitter/etc and then into Clubhouse in most cases.